src/modules/authentification/components/AuthLoginForm.tsx

```diff
@@ -1,48 +1,131 @@
+        });
+
+        setIsLoggedIn(true);
+        navigate("/home");
+      })
+      .catch((err) => {
+        throw new Error(err.message);
+      });
+  };
 
   return (
     <FormProvider {...form}>
-      <form>
-        <h2>Log In</h2>
-        <label>E-mail address</label>
-        <InputField
-          name="email"
-          placeholder="Enter your email..."
-          type="email"
-          validation={{ required: true }}
-        />
-        <label>Password</label>
-        <InputField
-          name="password"
-          placeholder="Enter a password..."
-          type="password"
-          validation={{
-            required: true,
-            minLength: {
-              value: passwordMinLength,
-              message: `Password must be atleast ${passwordMinLength} characters long.`,
-            },
-            maxLength: {
-              value: passwordMaxLength,
-              message: `Password cannot be longer than ${passwordMaxLength} characters.`,
-            },
-            pattern: {
-              value: passwordRegexp,
-              message:
-                "Password must contain atleast one uppercase letter, one lowercase letter and a number or a special character.",
-            },
-          }}
+      <form
+        id="log-in-form"
+        onSubmit={form.handleSubmit(onSubmit)}
+        className="relative flex flex-col w-screen sm:w-[500px] bg-basic-white text-text-default-gray py-8 px-10 sm:shadow-md"
+      >
+        <h2 className="form-size-header font-bold">Sign In</h2>
+        <span className="text-sm">
+          Don't have an account?{" "}
+          <Link to="/auth/register" className="text-text-purple">
+            Create one now!
+          </Link>
+        </span>
+        <section className="flex flex-col form-vertical-margin form-bottom-padding">
+          <label>E-mail address</label>
+          <InputField
+            name="email"
+            placeholder="Enter your email..."
+            type="email"
+            validation={{ required: true }}
+          />
+          <label>Password</label>
+          <InputField
+            name="password"
+            placeholder="Enter your password..."
+            type="password"
+            validation={{
+              required: true,
+              minLength: {
+                value: passwordMinLength,
+                message: `Password must be atleast ${passwordMinLength} characters long.`,
+              },
+              maxLength: {
+                value: passwordMaxLength,
+                message: `Password cannot be longer than ${passwordMaxLength} characters.`,
+              },
+              pattern: {
+                value: passwordRegexp,
+                message:
+                  "Password must contain atleast one uppercase letter, one lowercase letter and a number or a special character.",
+              },
+            }}
+          />
+        </section>
+        <PrimaryButton label="Sign In" variant="primary" />
```

Comment:
If the name is PrimaryButton, it shouldn't have variants "primary" or "secondary". Name the PrimaryButton just Button.

Reply:
Changed name to Button.